src/metrics.js

```diff
@@ -12,7 +12,8 @@ var interval = 1000; // how often to refresh our measurement
 var cpuUsage;
 var gcLastRun;
 
-var customMetrics = {}
+var customMetrics = {};
+var customMetersMetrics = new Set();
```

Comment:
There is no way to get only the customMeterMetrics or the custom metrics in general, don't you want to add a method for that? if it is ok to get it just from the getAll methods so don't add one